File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.c
int removeDuplicates(int* A, int n) {
    int i=0;
    int j=1;
    
    while(j<n){
        if(A[j]==A[i]){
            j++;
        }else{
            i++;
            A[i]=A[j];
        }
    }
    return i+1;
    
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int map[] = new int[203];
        for(int val : nums)map[val+100]++;
        int k=0;
        for(int i=0;i<202;i++){
            if(map[i] >= 1){
                nums[k] = i-100;
                k++;
            }
        }

        return k;
    
    }
}